pkgs/applications/window-managers/weston/default.nix

```diff
@@ -1,10 +1,12 @@
 { lib, stdenv, fetchurl
 , meson, ninja, pkg-config, python3, wayland-scanner
-, cairo, dbus, lcms2, libdrm, libevdev, libinput, libjpeg, libxkbcommon, mesa
-, seatd, wayland, wayland-protocols, xcbutilcursor
+, cairo, dbus, libdrm, libevdev, libinput, libxkbcommon, mesa, seatd, wayland
+, wayland-protocols, xcbutilcursor
```

Comment:
For the future: Why not keep `, seatd, wayland` on the second line to keep the diff cleaner. I'd be fine with lines getting shorter as a rewrapping of multiple lines to 80 characters makes diffs so unreadable.

PS: I'm really starting to understand why we're switching to the one dependency per line style.